### PR TITLE
Finish static_pages

### PR DESCRIPTION
**Pull request by thiennv57**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
-</head>
-<body>
-
-<%= yield %>
-
-</body>
+  <head>
+    <title><%= full_title yield :title %></title>
+    <%= stylesheet_link_tag "application", media: "all",
+                                           "data-turbolinks-track": true %>
```
Comment (vietanhbkaaa) [inline app/views/layouts/application.html.erb]: indent

**Comment by VuongLong** [comment]
ok

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+body
+{
+  padding: 50px;
+}
```
Comment (ThuBM) [inline app/assets/stylesheets/custom.css.scss]: add new line

**Comment by ThuBM** [comment]
OK

**Pull request merged**